src/EVault/modules/Vault.sol

```diff
@@ -191,8 +191,9 @@ abstract contract VaultModule is IVault, Base, AssetTransfers, BalanceUtils {
         Shares shares = assets.toSharesDown(vaultCache);
         if (shares.isZero()) revert E_ZeroShares();
 
-        increaseBalance(vaultCache, receiver, account, shares, assets);
+        // skim performs equivalent order of operations as the deposit function, without assets being pulled from the user
```

Comment:
Let's kill this comment, I don't understand why anyone would care about this and it's just confusing.

Reply:
well, it was auditor's suggestion to have it... :)

Reply:
Yeah but it's more confusing than helpful IMO

Reply:
comment removed